src/renderer.zig

```diff
@@ -122,6 +123,7 @@ pub fn updateViewport(width: u31, height: u31, fov: f32) void {
 	lastHeight = @intFromFloat(@as(f32, @floatFromInt(height))*main.settings.resolutionScale);
 	lastFov = fov;
 	game.projectionMatrix = Mat4f.perspective(std.math.degreesToRadians(fov), @as(f32, @floatFromInt(lastWidth))/@as(f32, @floatFromInt(lastHeight)), zNear, zFar);
+	game.playerProjectionMatrix = Mat4f.perspective(std.math.degreesToRadians(65), @as(f32, @floatFromInt(lastWidth))/@as(f32, @floatFromInt(lastHeight)), zNear, zFar);
```

Comment:
This variable should be exclusively defined and assigned to in the itemdisplay.zig file.
Everything else just makes it harder to reason about where it came from.